src/metabase/lib/equality.cljc

```diff
@@ -18,7 +18,9 @@
    [metabase.mbql.util.match :as mbql.u.match]
    [metabase.shared.util.i18n :as i18n]
    [metabase.util.log :as log]
-   [metabase.util.malli :as mu]))
+   [metabase.util.malli :as mu]
+   #?@(:clj  ([clojure.core.memoize :as memoize]))
+   #?@(:cljs ([metabase.util.memoize :as memoize]))))
```

Comment:
These can and should be combined in a single `#?@(...)` expression:

```suggestion
   #?@(:clj  ([clojure.core.memoize :as memoize])
       :cljs ([metabase.util.memoize :as memoize]))))
```

Reply:
Actually on looking further, we should write `src/metabase/util/memoize.clj` that does `potemkin/import-vars` of the handful of public functions from `clojure.core.memoize`. Then we can simply `(:require [metabase.util.memoize :as memoize])` without having to juggle this.

Reply:
done